jake-brian/starter-code/server.js

```diff
@@ -0,0 +1,215 @@
+'use strict';
+
+const pg = require('pg');
+const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+const PORT = process.env.PORT || 3000;
+const app = express();
+const conString = process.env.DATABASE_URL || 'postgres://localhost:5432/kilovolt';// done: Don't forget to set your own conString
+const client = new pg.Client(conString);
+client.connect();
+client.on('error', function(error) {
+  console.error(error);
+});
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.static('./public'));
+
+app.get('/new', function(request, response) {
+  response.sendFile('new.html', {root: './public'});
+});
+
+app.get('/articles', function(request, response) {
+  // REVIEW: This query will join the data together from our tables and send it back to the client.
+  // DONE: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
+  client.query(`SELECT * FROM authors INNER JOIN articles ON authors.author_ID = articles.author_id; `)
+  .then(function(result) {
+    response.send(result.rows);
+  })
+  .catch(function(err) {
+    console.error(err)
+  });
+});
+
+app.post('/articles', function(request, response) {
```

Comment:
The insertRecord() method is not working, problems are in one of the query. Ensure the method is working by follow the instructions here https://github.com/codefellows-seattle-301d24/09-sql-joins-relations/blob/master/CRUD-testing.md 
(after insert refresh page the new article jibli jiber should appear first), get this working and re-submit for full credit.